.github/workflows/pr.yml

```diff
@@ -100,12 +100,27 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: "${{ env.GO_VERSION }}"
-      - name: Check out SPI's code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          repository: 'redhat-appstudio/service-provider-integration-operator'
-          path: 'spi'
+      - name: Checkout SPI Repository
```

Comment:
Do we need some developer documentation for this?

Reply:
afai understand it runs the spi-check against main or the branch with matching name and which has a PR. 
Maybe we could have something in the develop docs like:

If you are making simultaneous changes in RemoteSecret and SPI, this is how you ensure that compatibility check runs with the correct commit:
Name the pr in RemoteSecret "abc" and ensure there is a PR in SPI from the same branch name...